pulp_python/app/pypi/views.py

```diff
@@ -305,7 +320,7 @@ def retrieve(self, request, path):
         return Response(data=data)
 
 
-class UploadView(ViewSet, PackageUploadMixin):
+class UploadView(PackageUploadMixin, ViewSet):
```

Comment:
This looks like an unrelated change. Is it where the 200 -> 202 in the tests come from?
Are these unrelated bugfixes?

Reply:
This is me fixing the inheritance of the classes the to be in the correct order. The 200 -> 202 change comes from using the pulpcore `OperationPostponedResponse` instead of a normal response. It seems that twine doesn't mind whether it receives a 200 or 202.